sqlmesh/utils/jinja.py

```diff
@@ -103,7 +103,7 @@ def call_name(node: nodes.Expr) -> t.Tuple[str, ...]:
     if isinstance(node, nodes.Name):
         return (node.name,)
     if isinstance(node, nodes.Const):
-        return (f"'{node.value}'",)
+        return (f"'node.value'",)
```

Comment:
This is wrong. Please see the previous version. Would you mind adding a test case for this too please?